README.md

```diff
@@ -17,20 +17,16 @@ These dbt-core features include:
 3. __[Access](https://docs.getdbt.com/docs/collaborate/govern/model-access)__ - control the `access` level of models within groups
 4. __[Versions](https://docs.getdbt.com/docs/collaborate/govern/model-versions)__ - create and increment versions of particular models.
 
+Additionally, `dbt-meshify` automates the code development required to split a monolithic dbt project into component projects, or connect multiple pre-existing dbt projects using cross-project `ref`.
```

Comment:
wanna add two bullets for connect and split like we have above? feels like we're burying the lede a bit

Reply:
what do you mean "like we have above"? The list above is `dbt-core` features

Reply:
I think all i meant is that it would be nice to call out the split and connect feature sets as prominently as the above list, not that they specifically should be merged together. perhaps it makes more sense to list the commands instead of core features at all

Reply:
got it - updated to specify the mesh features